contracts/finance/payment_splitter/mod.rs

```diff
@@ -163,4 +166,14 @@ impl<T: PaymentSplitterStorage> PaymentSplitterInternal for T {
         self._emit_payee_added_event(payee, share);
         Ok(())
     }
+
+    default fn _release_all(&mut self) -> Result<(), PaymentSplitterError> {
+        let payees = self.get().payees.clone();
+
+        for account in payees.into_iter() {
+            self.release(account)?;
```

Comment:
Could you add please one more internal method `_release` that accepts `&AccountId` and use it to avoid cloning of the `payees` please?=)

Reply:
Ahhh ok. I didn't understand how I was supposed to remove clone when you said something about it before. I get it now.

Reply:
Tried every way I could think of to remove `clone()` from `payees `and pass `account` as a reference to an internal `_release` method, but I always ended up with move/ownership errors like having an immutable borrow and a mutable borrow at the same time. Finally figured out that `Vec<AccountId>` doesn't support `copy`, but `AccountId` does so I took advantage of that.